sdks/python/scripts/run_integration_test.sh

```diff
@@ -213,9 +218,13 @@ if [[ -z $PIPELINE_OPTS ]]; then
   # Install test dependencies for ValidatesRunner tests.
   # pyhamcrest==1.10.0 doesn't work on Py2.
   # See: https://github.com/hamcrest/PyHamcrest/issues/131.
-  echo "pyhamcrest!=1.10.0,<2.0.0" > postcommit_requirements.txt
-  echo "mock<3.0.0" >> postcommit_requirements.txt
-  echo "parameterized>=0.7.1,<0.8.0" >> postcommit_requirements.txt
+  if [[ -z $REQUIREMENTS_FILE ]]; then
+    echo "pyhamcrest!=1.10.0,<2.0.0" > requirements.txt
```

Comment:
I would avoid using `requirements.txt` as this is a common name, and potentially can be already created by a beam developer for other reasons, then, the script could overwrite this file.